pkgs/os-specific/linux/esdm_es/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, stdenv, fetchFromGitHub, kernel, kmod }:
+
+stdenv.mkDerivation rec {
+  pname = "esdm_es";
+  version = "unstable-2023-06-19-${kernel.version}";
+
+  src = fetchFromGitHub {
```

Comment:
I don't love that we fetch this twice because it then needs to be kept in sync with the package. However I don't really know how to improve it either.